### PR TITLE
rename ShardId to ShardOrd

### PR DESCRIPTION
**Pull request by mzhangmzz**
As proposed in https://github.com/near/NEPs/pull/241

### CODE REVIEW

File: chain/chain-primitives/src/error.rs [hunk 1]
```diff
@@ -180,7 +180,7 @@ pub enum ErrorKind {
     InvalidBalanceBurnt,
     /// Invalid shard id
     #[fail(display = "Shard id {} does not exist", _0)]
```
Comment (bowenwang1996) [inline chain/chain-primitives/src/error.rs:182]: maybe we should change this error message as well

**Review by bowenwang1996 — COMMENTED** [review]
LGTM. If the PR is ready for view, please mark it as such.

File: chain/chain-primitives/src/error.rs (hunk 1 above)
Comment (mzhangmzz) [reply chain/chain-primitives/src/error.rs:182]: It's failing some test and I am trying to fix that now

**Comment by mzhangmzz** [comment]
Will start a new one

**Pull request closed without merging**